[receiver\mongodb] Falky test TestScraperLifecycle

### Component(s)

receiver/mongodb

### What happened?

See https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3494329092/jobs/5850031320

```
=== RUN   TestScraperLifecycle
    scraper_test.go:57: 
        	Error Trace:	D:\a\opentelemetry-collector-contrib\opentelemetry-collector-contrib\receiver\mongodbreceiver\scraper_test.go:57
        	Error:      	"124.9565ms" is not less than "100ms"
        	Test:       	TestScraperLifecycle
        	Messages:   	component start and stop should be very fast
--- FAIL: TestScraperLifecycle (0.13s)
```

### Collector version

latest

### Environment information

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")


### OpenTelemetry Collector configuration

_No response_

### Log output

_No response_

### Additional context

_No response_

--- a/receiver/mongodbreceiver/scraper_test.go
+++ b/receiver/mongodbreceiver/scraper_test.go
@@ -54,7 +54,7 @@ func TestScraperLifecycle(t *testing.T) {
 	require.NoError(t, scraper.start(context.Background(), componenttest.NewNopHost()))
 	require.NoError(t, scraper.shutdown(context.Background()))
 
-	require.Less(t, time.Since(now), 100*time.Millisecond, "component start and stop should be very fast")
+	require.Less(t, time.Since(now), 200*time.Millisecond, "component start and stop should be very fast")
 }
 
 var (

